ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -30,10 +30,12 @@ export default class ImportWithSeedPhrase extends PureComponent {
   }
 
   parseSeedPhrase = (seedPhrase) => {
-    return seedPhrase
-      .trim()
-      .match(/\w+/g)
-      .join(' ')
+    const trimmed = seedPhrase.trim()
+    if (!trimmed) {
+      return ''
+    }
+
+    return trimmed.match(/\w+/g).join(' ')
```

Comment:
It looks like certain inputs would still result in this failing (e.g. `$`). Maybe the result of `match` should be checked too? To ensure it isn't `null`.

Reply:
Oh, good catch! I'll fix this up too!